src/client/themes/dark/index.ts

```diff
@@ -46,30 +88,29 @@ const darkTheme: DefaultTheme = {
     },
 
     vaultActionButton: {
-      background: 'transparent',
-      borderColor: '#6E6E6E',
-      color: '#6E6E6E',
-      disabledContrast: '1.5',
+      background: '#272727',
+      borderColor: 'transparent',
+      color: '#FFFFFF',
+      // disabledContrast: '1.5',
 
       selected: {
-        background: 'transparent',
+        background: '#272727',
         borderColor: '#FFFFFF',
         color: '#FFFFFF',
       },
     },
 
-    walletButton: {
-      background: '#006AE3',
-      color: '#202020',
-    },
+    // walletButton: {
+    //   background: '#006AE3',
+    //   color: '#202020',
+    // },
```

Comment:
Remove?

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6